puppets.js

```diff
@@ -70,10 +70,11 @@ const getAppleSplashScreenData = async browser => {
               landscape: { width: 0, height: 0 },
             },
           );
-        },
-      );
-    return scrapeSplashScreenDataFromHIGPage();
-  });
+        });
+      return scrapeSplashScreenDataFromHIGPage();
+    },
+    { selector: constants.APPLE_HIG_SPLASH_SCR_SPECS_DATA_GRID_SELECTOR },
+  );
 
   if (splashScreenData == null) {
```

Comment:
Could you add strict equality operator in here, please?

Reply:
Although your remark is not related to the changeset of this PR, it's a valid point and I did necessary changes as it might produce another bug. The data returned here is neither null or undefined, instead an empty array, introduced array.length check here.